Guides/Cost Optimization in the Cloud.md

```diff
@@ -39,6 +39,17 @@ Rightsizing is a term that means identifying and adjusting specific resources to
 
 After rightsizing your compute service, you can typically enable autoscaling to dynamically adjust resources up and down based on demand in your workload. This means that if demand is low, autoscaling will reduce the amount of resources provisioned allowing you to save money. Along with autoscaling, you will typically set high and low thresholds which should be based around your typical workload.
 
+### Using spot instances
+
+This is a type of VM instances that can be terminated at any time when resources need to be reallocated for other tasks. Because of this, spot VMs are offered at a much lower price with cost reductions of up to 90% as it makes use of unused compute capacity. Spot VMs are suitable for fault-tolerant, stateless and flexible workloads that can handle interruptions but don't need to be completed at a specific time. Examples of these types of workloads include CI/CD jobs or batch processing jobs and more but it's not recommended for other types of workloads. 
```

Comment:
Consider changing "instances" to "instance" to match the singular "type." -> This is a type of VM instance that can be terminated...

Reply:
Consider using "they" instead of "it" to maintain consistency in referring to spot VMs --> Because of this, spot VMs are offered at a much lower price, with cost reductions of up to 90%, as they make use of unused compute capacity.